cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter uses the Roles and Organization Management outlined in [sig-governance](https://github.com/spinnaker/governance/blob/master/governance.md).
+
+## Scope
+
+The Contributor Experience Special Interest Group (SIG) is responsible for improving the experience of those who contribute upstream to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues](https://github.com/spinnaker/spinnaker/issues), we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Projects & Services
+
+-  In accordance with the code of conduct, establish policies, standards, and procedures for the use, moderation, and management of all public platforms officially used by the project, including but not limited to:
+  - [GitHub Management](https://github.com/spinnaker)
+  - [Slack](https://join.spinnaker.io/)
+  - [Spinnaker](https://www.youtube.com/channel/UCcxQbw8kT1-FRhFhO2QCetg) YouTube channel
+  - Zoom
+  - Google Meet
+- Set up alerting automation for GitHub events, including new contributors, new issues, and new pull requests.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required, see [GitHub Management](https://github.com/spinnaker) for more detail
+- Own and execute events that are targeted to the Spinnaker contributor community, including:
```

Comment:
Items in this list seem to have direct, conflicting overlaps with the [Summit SIG](https://github.com/spinnaker/governance/tree/master/sig-summit). Is this SIG meant to replace the Summit SIG? Are the leads of the Summit SIG aware of this?

Reply:
To my knowledge this is not a replacement

Reply:
An important distinction here is **contributor** events, which to refers to a contributor day or hackathon event specifically for a project contributor audience. I think that the Spinnaker Summit, which historically has been a conference event, is out of scope. That event is going to be managed by the Linux Foundation events team going forward, in collaboration with the Summit SIG, which will help develop the speaker content program and make platform/program decisions with the LF team in the same way other CDF events are managed.  This SIG can cooperate with the Summit SIG as necessary to add contributor events to the Summit as desired.

IMO: Producing events is a lot of work. Breaking it up this way, by audience, will help us use open processes to to the work collaboratively across stakeholder organizations when planning events. This is LF methodology and prevents one company from dominating the decisions.

@michaelgalloway and @gcomstock, please let us know your thoughts as the co-leads of the SIG : )

Reply:
Hi @dnilasor,

I don't have much to add here. Producing events is a lot of work for sure, and my involvement with this year's summit was thin. I'm all in support. Thank you!